src/optimizer/query_to_operator_transformer.cpp

```diff
@@ -231,6 +232,10 @@ void QueryToOperatorTransformer::Visit(parser::InsertStatement *op) {
       catalog::Catalog::GetInstance()
           ->GetDatabaseObject(op->GetDatabaseName(), txn_)
           ->GetTableObject(op->GetTableName());
+  
+  if (target_table == nullptr)
```

Comment:
At this point, I don't think `target_table` can be null. Have you found a scenario where the expectation is false?

Reply:
I don't think so. Should I remove this check? Is this code reachable from some other part, apart from a query coming from the postgres front-end?

Reply:
@pmenon  Should I remove this check?

Reply:
@schedutron Yes, it should be safe to remove.

Reply:
Wasn't this going to be removed?

Reply:
@pervazea Yes, I'll do that in another commit.